File: produit.php
<?php
/********************************
    Fichier : produit.php
    Auteur : Joel Lapointe
    Fonctionnalité : WEB-02 - Gestion des commandes
    Date : 23 avril 2018

    Vérification :
    Date                Nom                 Approuvé
    ====================================================

    Historique de modifications :
    Date                Nom                 Description
    ======================================================

 ********************************/
    class Produit
    {
        public $nom;
        public $quantite;
        public $prix;

        /**
         * Produit constructor.
         * @param $nom
         * @param $quantite
         * @param $prix
         */
        public function __construct($nom, $quantite, $prix)
        {
            $this->nom = $nom;
            $this->quantite = $quantite;
            $this->prix = $prix;
        }

        /**
         * @return mixed
         */
        public function getNom()
        {
            return $this->nom;
        }

        /**
         * @param mixed $nom
         */
        public function setNom($nom)
        {
            $this->nom = $nom;
        }

        /**
         * @return mixed
         */
        public function getQuantite()
        {
            return $this->quantite;
        }

        /**
         * @param mixed $description
         */
        public function setQuantite($description)
        {
            $this->description = $description;
        }

        /**
         * @return mixed
         */
        public function getPrix()
        {
            return $this->prix;
        }

        /**
         * @param mixed $prix
         */
        public function setPrix($prix)
        {
            $this->prix = $prix;
        }



    }
?>

File: Android/Tamtam/app/src/main/java/tamtam/tamtam/produit.java
/****************************************
 Fichier : produit.java
 Auteur : Joel Lapointe
 Fonctionnalité :
 Date : 7 mai 2018

 Vérification :
 Date               Nom                   Approuvé
 =========================================================


 Historique de modifications :
 Date               Nom                   Description
 =========================================================

 ****************************************/

package tamtam.tamtam;

public class produit {

    String nom;
    float prix;
    String format;

    public produit(String nom, float prix, String format) {
        this.nom = nom;
        this.prix = prix;
        this.format = format;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public float getQuantite() {
        return prix;
    }

    public void setQuantite(float prix) {
        this.prix = prix;
    }

    public String getFormat() {
        return format;
    }

    public void setFormat(String format) {
        this.format = format;
    }
}
